packages/amplify-category-interactions/src/provider-utils/supported-services.ts

```diff
@@ -0,0 +1,183 @@
+export const servicesMetadata = {
+  Lex: {
+    samples: {
+      BookTrip: [
+        {
+          cancelMessage: 'Okay, I have cancelled your reservation in progress.',
```

Comment:
are these prompts still used somewhere?

Reply:
these samples seem to still be used but not as prompts: https://github.com/lazpavel/amplify-cli/blob/inquirer-to-prompts-interactions-category/packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js#L102